voluncheer/profiles/test_forms.py

```diff
@@ -110,3 +118,84 @@ def test_organization_must_submit_unique_email(self):
         form1.save(self)  # Submitting form 1
         form2 = VolunteerCreationForm(data=data2)
         self.assertFalse(form2.is_valid())
+
+
+class VolunteerChangeFormTest(TestCase):
+    """Test cases for Volunteer Change form"""
+
+    def setUp(self):
+        """See base class."""
+        self.form = VolunteerChangeForm()
+        self.user = Volunteer.objects.create(
+            user=User.objects.create(
+                email="luke@jedi.com",
+                password="NOOOOOOOOOOOOOOOOOOO",
+                type=UserType.VOLUNTEER,
+            ),
+            first_name="Luke",
+            last_name="Skywalker",
+            date_of_birth="1955-09-25",
+            description="I want to come with you to Alderaan.",
+        )
+        self.user.save()
+
+    def test_validation(self):
+        """Test volunteer profile update validation"""
+        self.assertFalse(self.user.first_name == "R2D2")
```

Comment:
Prefer using more accurate assertions, in this case `self.assertNotEqual`. Here and elsewhere.

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertNotEqual